### PR TITLE
Remove feature reshape

### PR DESCRIPTION
**Pull request by lorenaqendro**
FloatList does not accept a 2 dim array so removed the feature reshape and kept it flattened.
It's later reshaped to [wide, feature_dim] so it will ok as input shape for the model.

### CODE REVIEW

**Comment by lorenaqendro** [comment]
@zhezh can you have a look?

**Comment by zhezh** [comment]
thanks for contribution, but this line is to make sure the input is in correct shape and will not slow the app.

**Comment by zhezh** [comment]
there might be some change in new tensorflow api, but since I have switched to pytorch, I will not maintain this repo for new tensorflow version.
